java/src/main/java/com/google/devtools/build/lib/worker/BUILD.bazel

```diff
@@ -1,5 +1,7 @@
 package(default_visibility = ["//visibility:public"])
 
+load("@rules_java//java:defs.bzl", "java_library")
```

Comment:
let's load from https://github.com/bazelbuild/rules_java/blob/master/java/java_library.bzl instead

Reply:
I'm fine with either way, but is there official guidance on which path is more official? Loading from defs.bzl is more widespread in cs.opensource.google https://cs.opensource.google/search?q=load.*java_library&sq=

Reply:
A similar search in bazel-contrib: https://github.com/search?q=org%3Abazel-contrib+%22load%28%5C%22%40rules_java%22&type=code

Reply:
defs.bzl is the old convention; the per-rule .bzl reduces transitive loads. See [this doc comment](https://docs.google.com/document/d/1L1JFgjpZ7SrBinb24DC_5nTIELeYDacikcme-YcA7xs/edit?disco=AAAA_GneyNw) (and the whole doc actually)